user_docs/en/changes.t2t

```diff
@@ -29,6 +29,7 @@ What's New in NVDA
 - Sound split: (#12985, @mltony)
   - Allows splitting NVDA sounds in one channel (e.g. left) while sounds from all other applications in the other channel (e.g. right).

```

Comment:
I probably shouldn't comment on this here, but since it's in the diff ...

```suggestion
  - Allows splitting NVDA sounds into one channel (e.g. left) while sounds from all other applications are placed in the other channel (e.g. right).
```